flux-tests/tests/pos/surface/scrape00.rs

```diff
@@ -0,0 +1,31 @@
+#![feature(register_tool)]
+#![register_tool(flux)]
+#![feature(custom_inner_attributes)]
+#![flux::cfg(scrape_quals = "true")]
```

Comment:
I think we can write it without the quotes

```suggestion
#![flux::cfg(scrape_quals = true)]
```